runtime/wasm/src/module.rs

```diff
@@ -299,6 +305,32 @@ where
         Ok(None)
     }
 
+    /// function store.get(entity: string, id: string): void
+    fn store_get(
+        &self,
+        entity_ptr: AscPtr<AscString>,
+        id_ptr: AscPtr<AscString>,
+    ) -> Result<Option<RuntimeValue>, Trap> {
+        let store_key = StoreKey {
+            subgraph: self.subgraph.id.clone(),
+            entity: self.heap.asc_get(entity_ptr),
+            id: self.heap.asc_get(id_ptr),
+        };
+
+        // Retrieve an Entity from the store
+        let get_result = self
+            .store
+            .lock()
+            .unwrap()
+            .get(store_key)
+            .map(|entity| entity);
+        let get_result: HashMap<_, _> = match get_result {
+            Ok(entity) => entity.into(),
+            Err(_e) => return Err(host_error("Error getting entity".to_string())),
```

Comment:
It's usually nice to format the error `e` into the message.

Reply:
Store.get() currently returns `Result<Entity, ()>`, though that will be changing soon.  I've made a note to update this error handling in my store errors branch.